aptos-move/framework/aptos-framework/sources/voting.spec.move

```diff
@@ -0,0 +1,195 @@
+spec aptos_framework::voting {
+    spec module {
+        pragma verify = true;
+        pragma aborts_if_is_strict;
+        invariant exists<timestamp::CurrentTimeMicroseconds>(@aptos_framework);
```

Comment:
Two comments:
(1) This cannot be true. The genesis state is an empty state where no resource "exists". The correct invariant is `chain_status::is_operating() ==> exists<timestamp::CurrentTimeMicroseconds>(@aptos_framework);`.
(2) As a convention, we put this kind of "resource existence" invariant in the module that defines the resource. It's the `timestamp` module in this case. And, actually, it already has the invariant.